src/Uno.UI/Controls/Window.macOS.cs

```diff
@@ -44,8 +44,210 @@ public Window(CGRect contentRect, NSWindowStyle aStyle, NSBackingStore buffering
 			Delegate = new WindowDelegate(this);
 			_inputPane = InputPane.GetForCurrentView();
 			_inputPane.Window = this;
+
+			// Register the types that can be dragging into the window (effectively 'AllowDrop').
+			// This must be done before dragging enters the window or dragging methods won't be invoked.
+			// We register for all possible types and then confirm whether they
+			// are actually supported within the draggingEntered and draggingUpdated methods.
+			// This has a minor side effect in the standard UI (zoom effect) but is considered acceptable.
+			RegisterForDraggedTypes(new string[]
+				{
+					NSPasteboard.NSPasteboardTypeUrl,
+					NSPasteboard.NSPasteboardTypeTIFF,
+					NSPasteboard.NSPasteboardTypePNG,
+					NSPasteboard.NSPasteboardTypeHTML,
+					NSPasteboard.NSPasteboardTypeRTF,
+					NSPasteboard.NSPasteboardTypeRTFD,
+					NSPasteboard.NSPasteboardTypeFileUrl,
+					NSPasteboard.NSPasteboardTypeString
+
+					/* For future use
+					UTType.URL,
+					UTType.Image,
+					UTType.HTML,
+					UTType.RTF,
+					UTType.FileURL,
+					UTType.PlainText,
+					*/
+				});
+		}
+
+		/// <summary>
+		/// Method invoked when a drag operation enters the <see cref="NSWindow"/>.
+		/// </summary>
+		/// <remarks>
+		/// 
```

Comment:
All of these 'remarks' comments could be moved out into a separate document explaining how macOS drag/drop integration works. It might be useful to have this document for all platforms adding to it as new platforms are supported. Let me know if a separate document is desirable.

Reply:
Ok, of course in separate docs it would be expanded as an overview as well.